Geometry/TrackerCommonData/python/cmsExtendedGeometry2017XML_cfi.py

```diff
@@ -35,18 +35,18 @@
         'Geometry/TrackerCommonData/data/PhaseI/v2/pixfwdbladeInnerZminus.xml',
         'Geometry/TrackerCommonData/data/PhaseI/v2/pixfwdbladeOuterZplus.xml',
         'Geometry/TrackerCommonData/data/PhaseI/v2/pixfwdbladeOuterZminus.xml',
-        'Geometry/TrackerCommonData/data/PhaseI/v2/pixbarmaterial.xml', 
```

Comment:
Is it allowed to change the existing extended geometries?

Reply:
This is not a replacement of the existing scenario. The existing scenarios are in Geometry/CMSCommonData. This is a test scenario to check the xml files where all tracker related materials are removed from materials.xml so that their densities can be reduced or increased to make some systematic studies. Are you suggesting that we should name this file differently?

Reply:
Ok, as far as it is test scenario I think we can change it